package.json

```diff
@@ -84,6 +83,7 @@
     "json-loader": "^0.5.7",
     "lodash": "^4.17.20",
     "node-sass": "^4.14.1",
+    "playcanvas": "https://github.com/playcanvas/engine/archive/c6f750223e3c184b26995a1d9952f769811ca542.tar.gz",
```

Comment:
is this correct?

Reply:
It's including the sourcecode of playcanvas/engine rather than the npm build so the vector2 class can be imported directly.

Reply:
It's also locked down to the 1.46.4 commit to protect against future changes.